code/modules/mob/living/simple_animal/abnormality/teth/forsaken_murderer.dm

```diff
@@ -0,0 +1,73 @@
+/mob/living/simple_animal/hostile/abnormality/forsaken_murderer //designed to be a very forgiving and standard abnormality.
+	name = "forsaken murderer"
+	desc = "A unhealthy looking human bound in a full body straightjacket. His neck is broken and in the middle of his forehead is a old wound that refuses to heal."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "forsakenmurdererinert"
+	icon_living = "forsakenmurdererinert"
+	icon_dead = "forsakenmurdererdead"
+	del_on_death = FALSE
+	maxHealth = 540 //originally was 270
+	health = 540
+	rapid_melee = 1 //attack speed modifier. 2 is twice the normal.
+	move_to_delay = 5 //speed? lower is faster.
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1, WHITE_DAMAGE = 1, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 2)
+	melee_damage_lower = 4 //the lowest damage in regular attacks. Normal murderer is 2~4 so we double it.
```

Comment:
Might be too low of a damage.

Reply:
Changed it to 6 lower and 12 higher since the implication of 2~4 is a larger range of damage than Fragment who is 3~4.